python/dask_cudf/dask_cudf/tests/test_dataframe.py

```diff
@@ -0,0 +1,58 @@
+import pandas as pd
+
+import cudf
+import dask.dataframe as dd
+import dask_cudf
+
+import pytest
+
+
+def test_get_dummies(data):
+    df = pd.DataFrame({"A": ["a", "b", "c", "a", "z"], "C": [1, 2, 3, 4, 5]})
+    df["B"] = df["A"].astype('category')
+    df["A"] = df["A"].astype('category')
+    ddf = dd.from_pandas(df, npartitions=10)
+    dd.assert_eq(dd.get_dummies(ddf).compute(), pd.get_dummies(df))
+    gdf = cudf.from_pandas(df)
+    gddf = dask_cudf.from_cudf(gdf, npartitions=10)
+    dd.assert_eq(dd.get_dummies(ddf).compute(), dd.get_dummies(gddf),
+                 check_dtype=False)
+
+    df = pd.DataFrame({"A": ["a", "b", "c", "a", "z"],
```

Comment:
You should include one example with a decent amount of data and significant number of partitions. You've passed 10 in all of these, but I believe this won't actually create 10 partitions given the data size. Sometimes, wonky behavior can surface by going down a codepath that only exposes itself with enough partitions.

Reply:
Added one example of a large dataset across 100 partitions.